src/MergeContext.js

```diff
@@ -311,7 +318,9 @@ class MergeContext {
         const prAgeMs = new Date() - new Date(this._createdAt());
         if (prAgeMs < Config.votingDelayMin()) {
             this._log("in minimal voting period");
-            return Config.votingDelayMin() - prAgeMs;
+            this._approvalResult.params = {desc: "waiting for initial review", state: "pending"};
```

Comment:
This description may not match reality -- we may already have 10 reviews here. I think this check has to go lower so that we deemphasize the unimportant time aspect when there are not enough votes. In that lower position, the descriptions would become something like `waiting for fast track objections` and `waiting for more votes or a slow burner timeout`, depending on whether we already have sufficient or just necessary votes.